Sources/PluginLibrary/plugin.pb.swift

```diff
@@ -126,7 +126,7 @@ public struct Google_Protobuf_Compiler_CodeGeneratorRequest: SwiftProtobuf.Messa
 
   public func _protoc_generated_isEqualTo(other: Google_Protobuf_Compiler_CodeGeneratorRequest) -> Bool {
     if fileToGenerate != other.fileToGenerate {return false}
-    if (parameter != other.parameter) {return false}
+    if ((_parameter != nil || other._parameter != nil) && (_parameter == nil || other._parameter == nil || _parameter! != other._parameter!)) {return false}
```

Comment:
Yea, this is just what some of the existing code was doing, so I was just following that hoping there was reason for it, I'm fine changing it to something better.